packages/locale/lang/fa.ts

```diff
@@ -2,18 +2,27 @@ export default {
   name: 'fa',
   el: {
     breadcrumb: {
-      label: 'Breadcrumb', // to be translated
+      label: 'مسیر راهنما', // to be translated
```

Comment:
Please remove the suffix comment `// to be translated` if it's translated.

Reply:
done : https://github.com/element-plus/element-plus/pull/18290/commits/a4aa3291c9f7bb99d61aacd16df2b1aa6e0007ee